packages/@ember/-internals/glimmer/tests/integration/modifiers/on-test.js

```diff
@@ -32,18 +31,6 @@ moduleFor(
       );
     }
 
-    '@test SUPPORTS_EVENT_OPTIONS is correct (private API usage)'(assert) {
```

Comment:
We don't need this test anymore.
https://github.com/glimmerjs/glimmer-vm/pull/1535

From the implementation notes:
```
/*
  Internet Explorer 11 does not support `once` and also does not support
  passing `eventOptions`. In some situations it then throws a weird script
  error, like:
  ```
  Could not complete the operation due to error 80020101
  ```
  This flag determines, whether `{ once: true }` and thus also event options in
  general are supported.
*/
```